### PR TITLE
Apply admin template

### PR DESCRIPTION
**Pull request by minhquan4080**

### CODE REVIEW

File: app/assets/javascripts/AdminLTE/demo.js [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    }
+    if($('body').hasClass('layout-boxed')) {
+      $("[data-layout='layout-boxed']").attr('checked', 'checked');
+    }
+    if($('body').hasClass('sidebar-collapse')) {
+      $("[data-layout='sidebar-collapse']").attr('checked', 'checked');
+    }
+    
+  }
+})(jQuery, $.AdminLTE);
```
Comment (lhhieu89) [inline app/assets/javascripts/AdminLTE/demo.js]: Do we need this file?

Comment (minhquan4080) [inline app/assets/javascripts/AdminLTE/demo.js]: It need for reponsive

Comment (lhhieu89) [inline app/assets/javascripts/AdminLTE/demo.js]: Okie, because its name is `demo.js`

Comment (minhquan4080) [inline app/assets/javascripts/AdminLTE/demo.js]: you can rename file name :dancer:

**Pull request merged**